auth/kinit_client/kinit.c

```diff
@@ -561,12 +561,12 @@ k5_kinit(struct k_opts *opts, struct k5_data *k5)
         if (opts->verbose)
             fprintf(stderr, _("Initialized cache\n"));
 
-        ret = k5_cc_store_primary_cred(k5->ctx, mcc, &my_creds);
+        /*ret = k5_cc_store_primary_cred(k5->ctx, mcc, &my_creds);
```

Comment:
This can be uncommented now, since you are now building the krb5 libs yourself in the Dockerfile.

Reply:
Addressed in latest updated PR